Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  case Region::UNKNOWN_REGION:
```

Comment:
It already logs unknown country codes that's why I think it's interesting to have a log entry for a title_version that doesn't have a known region.

Regardless, you have a point, if we can't manage the string translation properly and it finally is a burden, it's not a good idea to change the region_letter value.